api.js

```diff
@@ -25,3 +25,29 @@ export const fetchAllComments = (article_id) => {
     return data;
   });
 };
+
+export const upVote = (article_id) => {
+  return ncNewsApi
+    .patch(`/articles/${article_id}`, {
+      inc_votes: 1,
+    })
+    .then((response) => {
+      console.log(response);
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
+
+export const downVote = (article_id) => {
+  return ncNewsApi
+    .patch(`/articles/${article_id}`, {
+      inc_votes: -1,
+    })
+    .then((response) => {
+      console.log(response);
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
```

Comment:
You have two functions that are identical here with a single difference of a number. I would recommend making the function more dynamic and having only one functioned used for both of these cases.

Is there also a reason the response is being console.log'ed here instead of being returned?